File: 0412-fizz-buzz/0412-fizz-buzz.js
/**
 * @param {number} n
 * @return {string[]}
 */
var fizzBuzz = function(n) {
    let fb = [];
    let i = 1;
    while(i <= n){
        if(i % 15 == 0){
            fb.push("FizzBuzz");
        }
        else if(i % 3 == 0){
            fb.push("Fizz");
        }
        else if(i % 5 == 0){
            fb.push("Buzz");
        }
        else{
            fb.push(i.toString());
        }
        i++;
    }
    return fb;
};

File: 0412-fizz-buzz/0412-fizz-buzz.cpp
class Solution {
public:
    vector<string> fizzBuzz(int n) {
        vector<string> arr;
        for(int i=1; i<=n; i++){
            string s = "";
            if(i%3 == 0)
                s.append("Fizz");
            if(i%5 == 0)
                s.append("Buzz");
            if(s.empty())
                s.append(to_string(i));
            arr.push_back(s);
        }
        return arr;
        /*
        vector<string> arr;
        for(int i=1; i<=n; i++){
            if(i%15 == 0)
                arr.push_back("FizzBuzz");
            else if(i%3 == 0)
                arr.push_back("Fizz");
            else if(i%5 == 0)
                arr.push_back("Buzz");
            else{
                string ch = to_string(i);
                arr.push_back(ch);
            }
        }
        return arr;
        */
    }
};